lib/modules/keyboardNav.js

```diff
@@ -1172,6 +1180,11 @@ function imageMove(deltaX, deltaY) {
 	ShowImages.move(mostVisible, deltaX, deltaY);
 }
 
+function videoToggleMute() {
+	const mostVisible = getMostVisibleElementInThingByQuery('.res-video-container', ASSERT);
+	ShowImages.toggleMute(mostVisible);
```

Comment:
I tried to do this in my initial impl but ran into a few hurdles involving how RES determines which entity is in focus; I wasn't too sure how to handle it, so if you have any further details that would be appreciated.

Also, would there be any benefit to leaving toggleMute in ShowImages rather than pulling it out to here? I was under the impression (from the rest of the file) that we tried to limit the amount of logic happening in keyboardNav.js and instead defer logic to the ShowImages module/etc, which is why my impl has it like that, but if it isn't needed, I'll pull the toggleMute logic up to here.